### PR TITLE
[Fix] update part of model links

### PR DESCRIPTION
**Pull request by ZCMax**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
update part of model links

## Modification
update part of model links

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 Note: We are going through large refactoring to provide simpler and more unified usage of many modules.
 
-The compatibilities of models are broken due to the unification and simplification of coordinate systems. For now, most models are benchmarked with similar performance, though few models are still being benchmarked.
+The compatibilities of models are broken due to the unification and simplification of coordinate systems. For now, most models are benchmarked with similar performance, though few models are still being benchmarked. In this version, we update part of model checkpoints after the coordinate systems have been refactored. See more details in the [Changelog](docs/en/changelog.md).
```
Comment (ZwwWayne) [inline README.md:32]: In this version, we update part of model checkpoints after the coordinate systems have been refactored. -> In this version, we update some of the model checkpoints after the refactor of coordinate systems.

File: docs/en/changelog.md [hunk 2]
```diff
@@ -5,7 +5,8 @@
 #### Compatibility
 
 - We migrate all the mmdet3d ops to mmcv and do not need to compile them when installing mmdet3d.
-- To fix the unprecise timestamp and optimize its saving method, we reformat the point cloud data during Waymo data conversion. The data conversion time is also optimized significantly by supporting parallel processing. Please re-generate KITTI format Waymo data if necessary. See more details in the [compatibility documentation](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/compatibility.md).
+- To fix the imprecise timestamp and optimize its saving method, we reformat the point cloud data during Waymo data conversion. The data conversion time is also optimized significantly by supporting parallel processing. Please re-generate KITTI format Waymo data if necessary. See more details in the [compatibility documentation](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/compatibility.md).
+- We update part of model checkpoints after the coordinate systems have been refactored. See more details in the #1369.
```
Comment (ZwwWayne) [inline docs/en/changelog.md:9]: part of model checkpoints after the coordinate systems have been refactored. -> some of model checkpoints after the refactor of coordinate systems. Please stay tuned for the release of the remaining model checkpoints.

Comment (ZwwWayne) [inline docs/en/changelog.md:9]: It could be better if we list the model that have been fully updated.

Comment (Tai-Wang) [reply docs/en/changelog.md]: #1369 may need a hyperlink for reference here.

**Pull request merged**